app/viewmodels/checkAnswers/notification/OtherIncomeOrGainSourceSummary.scala

```diff
@@ -33,7 +34,7 @@ object OtherIncomeOrGainSourceSummary  {
 
         SummaryListRowViewModel(
           key     = "whereDidTheUndeclaredIncomeOrGain.checkYourAnswersLabel",
-          value   = ValueViewModel(HtmlFormat.escape(answer).toString),
+          value   = ValueViewModel(HtmlContent(HtmlFormat.escape(answer))),
```

Comment:
Avoid `HtmlContent` where possible and use `Text` instead. e.g. `ValueViewModel(Text(answer))`
(same applies to other occurrences - with exception of the one embedding `<br>`)